src/components/tables/TableItems.tsx

```diff
@@ -514,3 +517,35 @@ export const UpdateRole = ({
     </MenuList>
   </Menu>
 );
+export const CopyTextContainer = ({
+  text,
+  children,
+}: {
+  text: string | null;
+  children: React.ReactNode;
+}) => {
+  const setCopiedValue = useCopyToClipboard();
```

Comment:
`setCopiedValue` naming seems misleading as useCopyToClipboard() exports a copy function. 

`copyText` should work better.